handshaker_test.go

```diff
@@ -225,8 +225,8 @@ func TestHandshaker(t *testing.T) {
 					t.Errorf("Client is not finished")
 				}
 				// there should be no `Finished` last retransmit from client
-				if cntClientFinishedLastRetransmit != 0 {
-					t.Errorf("Number of client finished last retransmit is wrong, expected: %d times, got: %d times", 0, cntClientFinishedLastRetransmit)
+				if cntClientFinishedLastRetransmit != 4 {
+					t.Errorf("Number of client finished last retransmit is wrong, expected: %d times, got: %d times", 4, cntClientFinishedLastRetransmit)
```

Comment:
Is it expected?

Reply:
@at-wat sorry I messed up, I merged the other PR before seeing this comment.

Yes I believe so! It is getting retransmitted from the remote now. Before I think it was detecting handshake complete and not doing a rtx.